core/tests/protractor/preferences.js

```diff
@@ -19,30 +19,38 @@
 var general = require('../protractor_utils/general.js');
 var users = require('../protractor_utils/users.js');
 var workflow = require('../protractor_utils/workflow.js');
+var PreferencesPage = require('../protractor_utils/PreferencesPage');
 
 describe('Preferences', function() {
+  var preferencesPage = null;
+  var EditorBox = 'editor';
```

Comment:
Don't do this, you're duplicating constants in multiple files. Instead have methods like preferencePage.toggleFeedbackEmailsCheckbox().

Reply:
done!!